src/doctopus/db/core.clj

```diff
@@ -0,0 +1,107 @@
+(ns doctopus.db.core
+  (:require [korma.db :refer [defdb postgres]]
+            [clojure.string :as string :refer [split-lines]]
+            [korma.core :refer :all]
+            [clj-time.coerce :refer [to-sql-time]]
+            [clj-time.core :as clj-time]
+            [camel-snake-kebab.core :refer [->snake_case_keyword ->kebab-case-keyword]]
+            [camel-snake-kebab.extras :refer [transform-keys]]
+            [doctopus.configuration :refer [server-config]]))
+
+(defn- now
+  []
+  "returns a string for 'now' in SQL Timestamp format"
+  (to-sql-time (clj-time/now)))
+
+(defn- add-date-fields
```

Comment:
This function is the only thing that feels weird to me. We *should* be able to lean on the DB for both created-on and updated-on fields; DBs are rull good at that. That said: I haven't yet been able to figure out a decent way to use the tools at hand to make that happen. Soooooooo. For now, I'm gonna put a to-do on my own list to figure that the hell out.